docs/metrics/ga.md

```diff
@@ -116,7 +116,6 @@ Here are the current events on the website as of this writing, grouped by their
 | Featured Experiment Button       | button click          | Manage Featured Button State  | [1, 2, 3, 4, 5, 10, 11, 13]|
 | Click Cancel on tour dialogue    | button click          | cancel tour                   | [1, 2, 3, 10, 11, 13]      |
 | Complete the tour                | button click          | complete tour                 | [1, 2, 3, 10, 11, 13]      |
-| Click Take Tour on tour dialogue | button click          | take tour                     | [1, 2, 3, 10, 11, 13]      |
```

Comment:
There isn't a take tour button on the tour dialogue, though I may be missing something here. cc/ @teonbrooks

Reply:
hmm...yeah, I agree, there isn't a take tour button, I don't know how I came to have that then. that's fine to delete